batch-jberet/src/main/java/org/wildfly/extension/batch/jberet/DefaultBatchEnvironment.java

```diff
@@ -76,4 +76,9 @@ public JobXmlResolver getJobXmlResolver() {
     public Properties getBatchConfigurationProperties() {
         throw BatchLogger.LOGGER.noBatchEnvironmentFound(WildFlySecurityManager.getCurrentContextClassLoaderPrivileged());
     }
+
+    @Override
+    public String getApplicationName() {
+        return null;
+    }
```

Comment:
This method should also throw the same exception as other methods.

Reply:
Changes done.